src/opnsense/service/conf/actions.d/actions_unbound.conf

```diff
@@ -68,9 +68,9 @@ message:Checking Unbound configuration
 [dnsbl]
 command:
     /usr/local/opnsense/scripts/unbound/blocklists.py &&
-    /usr/local/sbin/pluginctl -s unbound restart
+    /usr/local/opnsense/scripts/unbound/wrapper.py -b
 parameters:
-type:script
+type:script_output
 message:Updating Unbound DNSBLs
 description:Update Unbound DNSBLs
```

Comment:
@swhite2 @AdSchellevis due to review for 22.1.7 I saw that this is also used for a cron job due to the attached description. will it even continue to work? if not we need to split into dnsbl and dnsbl.cron ...